### PR TITLE
GMP Documentation, Alerts, and Filters: Velero

### PR DESCRIPTION
**Pull request by algchoo**
This PR includes:
* documentation
* alert definitions
* dashboard filters

### CODE REVIEW

File: integrations/velero/documentation.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. You can run the following checks to verify
+  that {{app_name_short}} is exposing Prometheus-format metrics.
+
+   *  To determine if {{app_name_short}} is exposing metrics, run the following command:
+
+      <pre class="devsite-click-to-copy">
+      kubectl -n velero port-forward deploy/velero 8085
+      curl localhost:8085/metrics
+      </pre>
```
[the remark this answers is not in the document]
Comment (mtabasko) [reply integrations/velero/documentation.yaml:19]: I would suggest the following. Ideally, these steps should be numbered. Can "deploy/velero" be replaced by "{{cluster_name}}" here, so it matches the sample query on line 44?)

 To verify that {{app_name_short}} is exposing metrics on the expected endpoint, do the following:
1. Set up port forwarding by using the following command:
  <pre class="devsite-click-to-copy">
  kubectl -n {{namespace_name}} port-forward deploy/velero 8085
  </pre>
1. Access the endpoint \`localhost:8085/metrics\` by using the browser or the \`curl\` utility in another terminal session.

File: integrations/velero/documentation.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
+exporter_type: included
+app_name_short: Velero
+app_name: {{app_name_short}}
+app_site_name: Velero
+app_site_url: https://velero.io/
+exporter_name: the elero exporter
```
[the remark this answers is not in the document]
Comment (mtabasko) [reply integrations/velero/documentation.yaml:6]: Based on line 8 in the "prometheus_metadata.yaml" file, should this be "the Velero Prometheus exporter"? (definitely need the capital "V" though! :)

File: integrations/velero/documentation.yaml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    endpoints:
+  + - port: metrics
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+  +     deploy: velero
+additional_podmonitoring_info: |
+  Ensure that the port and matchLabels match that of the {{app_name_short}} pods you wish to monitor.
```
Comment (mtabasko) [inline integrations/velero/documentation.yaml:41]: Ensure that the values of the \`port\` and \`matchLabels\` fields match those ....

Comment (mtabasko) [reply integrations/velero/documentation.yaml:41]: want

File: integrations/velero/documentation.yaml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  {{app_name_short}} installations include the `metrics` named port and label `deploy: velero` by default.
```
Comment (mtabasko) [inline integrations/velero/documentation.yaml:42]: This is really hard to parse. I think it means something like this:

include a \`port\' value of \`metrics'` and a \`deploy\` label with the value \`velero\`

It might be clearer to say "By default, {{app_name_short}} installations include a \`port\' value of \`metrics'` and a \`deploy`\l abel with the value \`velero\`."

**Review by mtabasko — CHANGES_REQUESTED** [review]

File: integrations/velero/documentation.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (yqlu) [reply integrations/velero/documentation.yaml]: Text LGTM, but let's number this as per mary's suggestion

File: integrations/velero/documentation.yaml (hunk 4 above)
Comment (yqlu) [reply integrations/velero/documentation.yaml]: Hmmm, my only slight pushback to @mtabasko is that we typically say "X is listening on port 8085" or "X exposes the `metrics` port", never "X has a `port` value of `metrics`"

- "By default, {{app_name_short}} installations have a port named `metrics`"
- "By default, {{app_name_short}} installations expose metrics on a port named `metrics`"

sounds more natural. But I don't feel super strongly about it.

**Review by yqlu — APPROVED** [review]
LGTM after outstanding comments on formatting

File: integrations/velero/documentation.yaml (hunk 4 above)
Comment (yqlu) [reply integrations/velero/documentation.yaml]: OK I see that we're using this phrasing e.g. in the Jenkins PR too (https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/425/commits/5c3e15f8b47c4bf5cdcd1c5daa2bf6253839bb70).

In that case, I don't mind keeping this as it is, and then if we do decide to tweak the wording, it's less churn to push something to all the affected configs at once as a follow-up

File: integrations/velero/documentation.yaml [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+minimum_exporter_version: v1.3.0
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is emitting
+  metrics on the expected endpoints, do the following:
+
+  1. Set up port forwarding by using the following command:
+
+  <pre class="devsite-click-to-copy">
+  kubectl -n velero port-forward {{cluster_name}} 8085
```
Comment (yqlu) [inline integrations/velero/documentation.yaml:18]: I think this was right before? Typically cluster_name refers to the GKE cluster name, does it have a different meaning in the context of velero?

Or at least, when I installed velero to test these instructions locally, I was using `kubectl -n {{namespace_name}} port-forward deploy/velero 8085`

**Review by yqlu — APPROVED** [review]
Just one question, then LGTM

File: integrations/velero/documentation.yaml (hunk 5 above)
Comment (algchoo) [reply integrations/velero/documentation.yaml]: That's my mistake, should be good to go.

**Review by yqlu — APPROVED** [review]
